.kokoro/tests/run_tests.sh

```diff
@@ -61,6 +61,8 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
     export LOCATION_ID=us-east1
     # For Datalabeling samples to hit the testing endpoint
     export DATALABELING_ENDPOINT="test-datalabeling.sandbox.googleapis.com:443"
```

Comment:
You are making this public, do you really want to do this?  Are customers using this?